Common/AlphaRuntimeStatistics.cs

```diff
@@ -60,12 +60,17 @@ public class AlphaRuntimeStatistics
         /// </summary>
         public decimal MeanPopulationEstimatedAlphaValue => TotalAlphasClosed > 0 ? TotalEstimatedAlphaValue / TotalAlphasClosed : 0;
 
+        /// <summary>
+        /// Base account currency
+        /// </summary>
+        public string AccountCurrency { get; internal set; }
```

Comment:
Instead of having a setter, the `ToDictionary` method could simply accept the account currency symbol. This can apply to other cases as well. If a method requires the configured account currency, it is sometimes best to add it as an argument. If the class makes judicious use of the account currency, then I would recommend it be a constructor argument. Having dangling properties that require consumers to set them at some point in the future will inevitably end with someone not initializing the object properly. This is why it's best to force the issue via a constructor argument or a method argument, as it forces the consumer to make a decision.